NonEmergencyAmbulanceTransportation/R4/files/NonEmergencyAmbulanceTransportationPrepopulation-0.1.0.cql

```diff
@@ -32,4 +32,35 @@ define "ServiceEndDate": FHIRHelpers.ToDateTime(
       (service_request.occurrence as FHIR.Period).end
     )
   )
+
+// TODO: Patient Conditions will need to be filtered by NEAT Condition Value once they have been created
+define RelevantDiagnoses: 
+  CodesFromConditions(Confirmed(ActiveOrRecurring([Condition]))) 
+
+define OtherDiagnoses:
+  CodesFromConditions(Confirmed(ActiveOrRecurring([Condition]))) except RelevantDiagnoses
+
+// Helpers
+
+// TODO: Evaluation Date will need to be replaced with the Evaluation Date subquestionnaire once it has been merged into master
+define "Today": Today()
```

Comment:
BasicClinicalInfo Library has Today defined if you need to use it.

Reply:
Will add that in! Thanks!